app/views/calculation/whatNext/whatNextSAGain.scala.html

```diff
@@ -41,7 +41,7 @@ <h1 class="govuk-heading-xl">@Messages("calc.yourOptions.title")</h1>
 
         <p id="report-now-information" class="govuk-body">@Messages("calc.whatToDoNext.gain.reportNowInfo")</p>
 
-        <a id="report-now-button" class="govuk-button" href="@iFormUrl">@Messages("calc.whatToDoNext.reportNow")</a>
+        <a id="report-now-button" class="govuk-button" data-module="govuk-button" role="button" href="@iFormUrl">@Messages("calc.whatToDoNext.reportNow")</a>
```

Comment:
Also better to use GovukButton:
```suggestion
        @govukButton(Button(
            href = iFormUrl, 
            content = Text(messages(""calc.whatToDoNext.reportNow"")),
            attributes = Map("id" -> "report-now-button")
        ))
```